templates/status/judge-status-table.jade

```diff
@@ -24,17 +24,17 @@ for judge in judges
             if judge.online
                 | #{judge.uptime|timedelta:"simple"}
             else
-                | N/A
+                | {% trans "N/A" %}
         td.ping
-            if judge.online
+            if judge.online and judge.ping_ms
                 | #{judge.ping_ms|floatformat:3} ms
```

Comment:
Ideally this will also be translated, but we'll just say ms is the SI unit and must therefore be international.